common/ethnicities/wc_ethnicities_gnome.txt

```diff
@@ -0,0 +1,401 @@
+        1 = { name = head_top_height_neg    range = { 0 0.33  }    } # REMEMBER TO ADJUST RANGE!
+	}
+	gene_head_top_width = {
+        1 = { name = head_top_width_neg    range = { 1 1 }    } # REMEMBER TO ADJUST RANGE!
+	}
+	
+	#Eyes
+	special_eyes = {
+        10 = { name = no_special_eyes      range = { 0 1 } }
+    }
+	gene_eye_depth = {
+        1 = { name = eye_depth_neg    range = { 0 0.45  }    }
+	}
+	gene_eye_angle = {
+        1 = { name = eye_angle_neg    range = { 0.4 0.4  }    }
+	}
+	gene_eye_height = {
+        1 = { name = eye_height_neg    range = { 0.2 0.4  }    }
+	}
+	gene_eye_distance = {
+        1 = { name = eye_distance_neg    range = { 0.4 0.5  }    }
+	}
+	gene_eye_shut = {
+        1 = { name = eye_shut_neg    range = { 0.35 0.5  }    }
+	}
+	gene_bs_eye_corner_depth = {
+        1 = { name = eye_corner_depth_neg    range = { 0 1 }    }
+		1 = { name = eye_corner_depth_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_fold_shape = {
+        1 = { name = eye_fold_shape_pos    range = { 0 1 }    }
+		1 = { name = eye_fold_shape_neg    range = { 0 0.5 }    }
+	}
+	gene_bs_eye_size = {
+        1 = { name = eye_size_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_upper_lid_size = {
+        1 = { name = eye_upper_lid_size_pos    range = { 0.5 1 }    }
+	}
+	face_detail_eye_lower_lid_def = {
+        1 = { name = eye_lower_lid_def    range = { 0.5 1 }    }
+	}
+	face_detail_eye_socket = {
+        1 = { name = eye_socket_01    range = { 0 1 }    }
+		1 = { name = eye_socket_02    range = { 0 1 }    }
+		1 = { name = eye_socket_03    range = { 0 1 }    }
+	}
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	
+	#Eyebrows
+	gene_eyebrows_shape = {
+		1 = {  name = no_eyebrows             range = { 0 1 }     }
+        1 = {  name = avg_spacing_lower_thickness             range = { 0 1 }     }
+		1 = {  name = close_spacing_lower_thickness             range = { 0 1 }     }
+		1 = {  name = far_spacing_lower_thickness             range = { 0 1 }     }
+	}
+	gene_eyebrows_fullness = {
+        1 = {  name = no_eyebrows             range = { 0 1 }     }
+        1 = {  name = layer_2_avg_thickness             range = { 0 1 }     }
+        1 = {  name = layer_2_low_thickness             range = { 0 1 }     }
+	}
+	
+	
+	###Other	
+	hairstyles = {
+		1 = { name = western_hairstyles 		range = { 0 0.34 } }
+		1 = { name = western_hairstyles 		range = { 0.64 0.78 } }
+		1 = { name = mena_hairstyles 		range = { 0 1 } } # very good for male gnomes, not so much for females
+		1 = { name = byzantine_hairstyles 		range = { 0.82 0.82 } } #male_hair_byzantine_02
+	}
+	beards = {
+		1 = { name = western_beards 		range = { 0.49 0.92 } }
+		1 = { name = mena_beards 		range = { 0.40 0.74 } }
+		1 = { name = steppe_beards 		range = { 0 1 } }
+		1 = { name = sub_saharan_beards 		range = { 0.92 0.92 } } #male_beard_sub_saharan_03
+		1 = { name = indian_beards 		range = { 0 1 } }
+	}
```

Comment:
Create beard and hair genes here.

Reply:
> 
> 
> Create beard and hair genes here.

Added in 6b389930b94a7a178d0bb1f07d2d6dcc0b76daab.